src/app/containers/FrontPage/index.stories.jsx

```diff
@@ -28,28 +28,37 @@ const serviceDatasets = {
   punjabi: punjabiData,
 };
 
+// eslint-disable-next-line react/prop-types
+const DataWrapper = ({ service, children }) => {
```

Comment:
This is here as a way to provide async data too the story. Storybook (as far as i can find) doesnt support async stories so you have to do this

Reply:
Could this potentially cause issues with chromaticqa tests if we are async loading data

Reply:
why did the preprocessor function become an async function?

Reply:
It had to because 1 preprocessor function is async (the xml one). So rather than have every rule handle async data, I made it so the preprocessor itself if async, and passes sync data to each preprocessor